Dockerfile no longer builds

```
Step 13/30 : RUN make build
 ---> Running in af9395fcb7ef
make: *** No rule to make target 'build'.  Stop.
Removing intermediate container af9395fcb7ef
The command '/bin/sh -c make build' returned a non-zero code: 2
```

Looks like the build target went away in https://github.com/oragono/oragono/commit/b6feca05a335852142b1fb41cafeb2a1ec4a05db

@slingamn what target should this be using now? or should the dockerfile just call `goreleaser --snapshot --rm-dist`?

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,15 +3,6 @@ FROM golang:rc-alpine AS build-env
 
 RUN apk add --no-cache git make curl
 
-# install goreleaser
-RUN mkdir -p /go/src/github.com/goreleaser
-WORKDIR /go/src/github.com/goreleaser
-
-RUN git clone https://github.com/goreleaser/goreleaser.git
-WORKDIR /go/src/github.com/goreleaser/goreleaser
-RUN make setup build
-RUN cp ./goreleaser /usr/bin
-
 # get oragono
 RUN mkdir -p /go/src/github.com/oragono
 WORKDIR /go/src/github.com/oragono
@@ -20,7 +11,7 @@ RUN git clone --recurse-submodules https://github.com/oragono/oragono.git
 WORKDIR /go/src/github.com/oragono/oragono
 
 # compile
-RUN make build
+RUN make
 
 
 
@@ -41,7 +32,7 @@ EXPOSE 6667/tcp 6697/tcp
 
 # oragono itself
 RUN mkdir -p /ircd-bin
-COPY --from=build-env /go/src/github.com/oragono/oragono/dist/linux_amd64/oragono /ircd-bin
+COPY --from=build-env /go/bin/oragono /ircd-bin
 COPY --from=build-env /go/src/github.com/oragono/oragono/languages /ircd-bin/languages/
 COPY --from=build-env /go/src/github.com/oragono/oragono/oragono.yaml /ircd-bin/oragono.yaml
 COPY run.sh /ircd-bin/run.sh

